ui/src/timeMachine/actions/queries.ts

```diff
@@ -76,7 +76,14 @@ export const getOrgIDFromBuckets = (
   text: string,
   allBuckets: Bucket[]
 ): string | null => {
-  const ast = parse(text)
+  let ast: any
```

Comment:
did it throw a compiler warning not having `: any` here? typing something as `any` is kind of implied and so i don't think we do it very often.

Reply:
actually I meant to remove this, so good catch :)